CHANGELOG.md

```diff
@@ -5,47 +5,47 @@
 
 #### Behavioral Changes
 - Add target to Docker image names for studio [#5805](https://github.com/habitat-sh/habitat/pull/5805) ([smacfarlane](https://github.com/smacfarlane))
+- Move env var and default value for listen gossip to the clap layer [#5884](https://github.com/habitat-sh/habitat/pull/5884) ([mpeck](https://github.com/mpeck))
```

Comment:
Looks like the changelog wasn't properly updated for 0.69.0 after that release (bumping the version in the changelog is currently manual).

We should fix that before proceeding further.